src/services/EmailService.js

```diff
@@ -0,0 +1,53 @@
+import NodeMailer from 'nodemailer';
+
+class EmailService {
+  constructor(smtpHost, smtpUsername, smtpPassword, smtpPort, smtpTls) {
+    this.smtpHost = smtpHost;
+    this.smtpUsername = smtpUsername;
+    this.smtpPassword = smtpPassword;
+    this.smtpPort = smtpPort;
+    this.smtpTls = smtpTls;
+  }
+
+  _smtpConnect = () => new Promise((resolve, reject) => {
+    // create reusable transporter object using the default SMTP transport
+    try {
+      resolve(NodeMailer.createTransport({
+        host: this.smtpHost,
+        port: this.smtpPort,
+        secure: this.smtpTls,
+        auth: {
+          user: this.smtpUsername,
+          pass: this.smtpPassword,
+        },
+      }));
+    } catch (err) {
+      reject(err);
+    }
+  });
+
+  sendEmail = async req => this._smtpConnect()
```

Comment:
this method should not try to send an email. It should just accept the request and have a try/catch that returns a success or failure response to the caller. We will fill in the remaining functionality in future stories.

Reply:
done